src/style.css

```diff
@@ -0,0 +1,112 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url("fonts/Avenir-Book.ttf")
+    format("truetype");
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  align-items: center;
+  flex-direction: column;
+  position: absolute;
+  margin-top: 20px;
+  width: 100%;
+}
+
+.search-bar__big {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: calc(100% - 18px);
+  height: 68px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 3px;
+  background-color: #fff;
+  color: #3d4e61;
+}
+
+.search-bar__small {
+  display: flex;
+  align-items: center;
+  position: relative;
+  width: calc(100% - 18px);
+  margin-top: 20px;
+  height: 40px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  border-radius: 3px;
+  border: 1px solid #e1e7ed;
+  margin-bottom: 10px;
+  color: #3d4e61;
+}
+
+.search-bar__big input[type="text"] {
+  height: 19px;
+  line-height: 19px;
+  width: 129px;
+  margin-left: 18px;
+  font-size: 16px;
+  letter-spacing: 0;
+  color: #3d4e61;
+  border-color: transparent;
+  font-family: "Avenir-Book",
+    sans-serif;
+  text-align: left;
+  margin-bottom: 2px;
+}
+
+.search-bar__small input[type="text"] {
+  width: 158px;
+  height: 19px;
+  margin-left: 7px;
+  font-size: 14px;
+  line-height: 19px;
+  letter-spacing: 0;
+  color: #3d4e61;
+  border-color: transparent;
+  font-family: "Avenir-Book",
+    sans-serif;
+  text-align: left;
+  margin-bottom: 2px;
+}
```

Comment:
you can try to combine some similar styles into another className, and add this className to each block